airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumRecordIterator.kt

```diff
@@ -174,7 +174,8 @@ class DebeziumRecordIterator<T>(
 
             // #41647: discard event type with op code 'm'
             if (!isEventTypeHandled(changeEventWithMetadata)) {
-                LOGGER.info { "WAL event type not handled: $next" }
+                LOGGER.info { "WAL event type not handled: ${changeEventWithMetadata.eventValueAsJson?.get("op")?.asText()}" }
```

Comment:
Can we remove this log as a whole? This will also make the the to potentially increase a lot. The metric we have are events and I wonder if GCP is billing per event rather than per size of the log.

Reply:
This log is useful as it indicates we're getting from debezium something we do not expect -- something is very wrong. 
Case in point is Abacus ([oncall/#5335](https://github.com/airbytehq/oncall/issues/5335)).
Would this amount of extra log will be too much with this change?
Each log line is going to be "WAL event type not handled: 'r'"

Reply:
I think that it will be better. I was probably wrong about the pricing details, it seems to be per Gb only https://cloud.google.com/logging#pricing though.